src/Renderer/ThreeExtented/GlobeControls.js

```diff
@@ -44,51 +46,152 @@ var CONTROL_KEYS = {
     S: 83,
 };
 
+// TODO: can be optimize for some uses
+var presiceSlerp = function (qb, t) {
```

Comment:
Three.js `Quaternion.slerp` considers an angle less than 0.001 to zero ([see](https://github.com/mrdoob/three.js/blob/master/src/math/Quaternion.js#L443))